tree/dataframe/inc/ROOT/RDF/RMetaData.hxx

```diff
@@ -32,10 +32,20 @@ namespace Experimental {
 /**
 \class ROOT::RDF::Experimental::RMetaData
 \ingroup dataframe
-\brief Class behaving as a heterogenuous dictionary to store dataset metadata
+\brief Class behaving as a heterogenuous dictionary to store the metadata of a dataset.
 
- This class should be passed to an RSample object which represents a single dataset sample.
- Once a dataframe is built with RMetaData object, it could be accessed via DefinePerSample.
+ An example of creating the RMetaData object:
+ ~~~{.cpp}
+ ROOT::RDF::Experimental::RMetaData meta;
+ meta.Add("sample_name", "name"");
+ meta.Add("luminosity", 10064);
+ meta.Add("xsecs", 1.0);
+ ~~~
+
```

Comment:
Maybe we could also add a comment here specifying that the only supported types in RMetaData are `std::string`,`int`,`double`